### ISSUE TITLE
Incomplete HTML output in test data and real data.

### ISSUE THREAD

**Issue by jamesdalg** [aryeelab/hichipper#26]
bash-4.1$ hichipper --out output_real yaml/one.yaml
Thu Oct 26 16:53:47 EDT 2017: Starting hichipper pipeline v0.7.0
Thu Oct 26 16:53:47 EDT 2017: Parsing user parameters
Anchors removed due to excessive size (likely at ends of chromosomes): 0
Thu Oct 26 16:53:50 EDT 2017: Processing d1
Thu Oct 26 16:53:50 EDT 2017: Total_PETs=164332898
Thu Oct 26 16:53:51 EDT 2017: Mapped_unique_quality_pairs=1888170
Thu Oct 26 16:53:51 EDT 2017: Mapped_unique_quality_valid_pairs=627403
Thu Oct 26 16:53:51 EDT 2017: Intersecting PETs with anchors
Thu Oct 26 16:53:51 EDT 2017: Finished the anchor merging.
Thu Oct 26 16:54:23 EDT 2017: Intrachromosomal_valid_small=163462
Thu Oct 26 16:54:24 EDT 2017: Intrachromosomal_valid_med=394048
Thu Oct 26 16:54:25 EDT 2017: Intrachromosomal_valid_large=69902
Thu Oct 26 16:54:25 EDT 2017: Total number of anchors used: 926
Thu Oct 26 16:54:25 EDT 2017: Total number of reads in anchors: 410414
Thu Oct 26 16:54:31 EDT 2017: Mapped_unique_intra_quality_anchor=20288
Thu Oct 26 16:54:31 EDT 2017: Mapped_unique_intra_quality_anchor_small=13632
Thu Oct 26 16:54:31 EDT 2017: Mapped_unique_intra_quality_anchor_med=6090
Thu Oct 26 16:54:31 EDT 2017: Mapped_unique_intra_quality_anchor_large=566
Thu Oct 26 16:54:31 EDT 2017: Loop_PETs=6090
Thu Oct 26 16:54:31 EDT 2017:
['Rscript', u'output_real/qcReport.R', '/usr/local/Anaconda/envs_app/hichipper/0.7.0/lib/python2.7/site-packages/hichipper', 'output_real', '/spin1/users/dalgleishjl/hichipper/tests', '0.7.0', 'd1']
/spin1/users/dalgleishjl/hichipper/tests

processing file: qcReport_make.Rmd
  |...                                                              |   5%
   inline R code fragments

  |.......                                                          |  11%
label: unnamed-chunk-1 (with options)
List of 3
 $ echo   : logi FALSE
 $ message: logi TRUE
 $ warning: logi TRUE

  |..........                                                       |  16%
   inline R code fragments

  |..............                                                   |  21%
label: unnamed-chunk-2 (with options)
List of 3
 $ echo   : logi FALSE
 $ message: logi FALSE
 $ warning: logi FALSE

  |.................                                                |  26%
  ordinary text without R code

  |.....................                                            |  32%
label: unnamed-chunk-3 (with options)
List of 4
 $ echo   : logi FALSE
 $ message: logi FALSE
 $ warning: logi FALSE
 $ results: chr "asis"

  |........................                                         |  37%
  ordinary text without R code

  |...........................                                      |  42%
label: unnamed-chunk-4 (with options)
List of 4
 $ echo     : logi FALSE
 $ message  : logi FALSE
 $ warning  : logi FALSE
 $ out.width: chr "\\textwidth"

  |...............................                                  |  47%
  ordinary text without R code

  |..................................                               |  53%
label: unnamed-chunk-5 (with options)
List of 4
 $ echo     : logi FALSE
 $ message  : logi FALSE
 $ warning  : logi FALSE
 $ out.width: chr "\\textwidth"

  |......................................                           |  58%
  ordinary text without R code

  |.........................................                        |  63%
label: unnamed-chunk-6 (with options)
List of 6
 $ echo     : logi FALSE
 $ message  : logi FALSE
 $ warning  : logi FALSE
 $ results  : chr "asis"
 $ out.width: chr "\\textwidth"
 $ fig.width: num 7

  |............................................                     |  68%
  ordinary text without R code

  |................................................                 |  74%
label: unnamed-chunk-7 (with options)
List of 4
 $ echo     : logi FALSE
 $ message  : logi FALSE
 $ warning  : logi FALSE
 $ out.width: chr "\\textwidth"

Using count as value column: use value.var to override.
  |...................................................              |  79%
  ordinary text without R code

  |.......................................................          |  84%
label: unnamed-chunk-8 (with options)
List of 4
 $ echo     : logi FALSE
 $ message  : logi FALSE
 $ warning  : logi FALSE
 $ out.width: chr "\\textwidth"

  |..........................................................       |  89%
  ordinary text without R code

  |..............................................................   |  95%
label: unnamed-chunk-9 (with options)
List of 4
 $ echo     : logi FALSE
 $ message  : logi FALSE
 $ warning  : logi FALSE
 $ out.width: chr "\\textwidth"

  |.................................................................| 100%
  ordinary text without R code

output file: qcReport_make.knit.md

/usr/local/apps/pandoc/1.15.0.6/bin/pandoc +RTS -K512m -RTS qcReport_make.utf8.md --to html --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash --output output_real.hichipper.qcreport.html --smart --email-obfuscation none --self-contained --standalone --section-divs --template /usr/local/apps/R/gcc_4.9.1/library/rmarkdown/rmd/h/default.html --highlight-style pygments --css /usr/local/apps/R/gcc_4.9.1/library/rmarkdown/rmarkdown/templates/html_vignette/resources/vignette.css --include-in-header /spin1/scratch/dalgleishjl/RtmpR3VvG1/rmarkdown-str62f25b19add6.html --mathjax --variable 'mathjax-url:https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML'
output1 is a zipped copy of the results for test data and seems to have an html file that doesn't have complete output.

Output created: output_real.hichipper.qcreport.html
Warning messages:
1: Transformation introduced infinite values in continuous x-axis
2: Removed 13419 rows containing non-finite values (stat_bin).
Processing: d1
[output1.zip](https://github.com/aryeelab/hichipper/files/1420060/output1.zip)

**Comment by caleblareau** [comment]
Hi @jamesdalg -- two things

1) when calling loop anchors, see that you only have 926, which leads to a very small number of loops. This is a problem that I've experienced for some HiChIP datasets newer than those described by Mumbach et al. but I haven't found an immediate solution. 

2) The incomplete output that you seem to be referring to is a function of the sankey diagrams and DT tables not rendering properly. It's hard to know what's causing this with your current output. Can you verify that all the R package dependencies are up to date? What versions of the packages and R are you using?

**Comment by caleblareau** [comment]
Hi @jamesdalg I've modified the tool such that in version `0.7.1`, I use less flashy tools to render tables, figures, etc., which I'm hoping will fix this. Let me know if not.
